x/datapool/client/cli/queryPool.go

```diff
@@ -119,3 +119,47 @@ func CmdGetDataValidationCertificates() *cobra.Command {
 
 	return cmd
 }
+
+func CmdGetDataPassRedeemReceipts() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "data-pass-redeem-receipts [poolID] [redeemer]",
+		Short: "Query data pass redeem receipts by pool and redeemer",
+		Args:  cobra.ExactArgs(2),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clientCtx, err := client.GetClientQueryContext(cmd)
+			if err != nil {
+				return err
+			}
+
+			queryClient := types.NewQueryClient(clientCtx)
+
+			pageReq, err := client.ReadPageRequest(cmd.Flags())
+			if err != nil {
+				return err
+			}
+
+			poolID, err := strconv.ParseUint(args[0], 10, 64)
+			if err != nil {
+				return err
+			}
+
+			redeemer := args[1]
+
+			res, err := queryClient.DataPassRedeemReceipts(cmd.Context(), &types.QueryDataPassRedeemReceiptsRequest{
+				PoolId:     poolID,
+				Redeemer:   redeemer,
+				Pagination: pageReq,
+			})
+
+			if err != nil {
+				return err
+			}
+
+			return clientCtx.PrintProto(res)
+		},
+	}
+
+	flags.AddPaginationFlagsToCmd(cmd, "all data pass")
```

Comment:
Please add query flags
```suggestion
	flags.AddPaginationFlagsToCmd(cmd, "all data pass")
	flags.AddQueryFlagsToCmd(cmd)
```

Reply:
This will be fixed in #322 branch.